xcodeproj/internal/xcodeproj_macro.bzl

```diff
@@ -287,6 +288,18 @@ def xcodeproj(
               custom schemes are provided.
             </li>
             </ul>
+        scheme_autogeneration_config: Optional. A struct returned by
+            `xcschemes.autogeneration_config`.
+
+            Allows further configuration of `scheme_autogeneration_mode`.
+
+            Available configurations:
+
+            <ul>
+            <li>
+              `scheme_name_exclude_patterns`: Used to skip creating schemes matching provided regex patterns.
+            </li>
+            </ul>
```

Comment:
To prevent doc duplication, this should only be detailed in `xcschemes.autogeneration_config`.

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/3c0bce7a2c8826fb7a3d511dfdbfa744e3efd5a2